go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
 	github.com/hashicorp/raft v2.0.0-20200420012049-88ad3b3f0a54+incompatible
 	github.com/hashicorp/raft-boltdb v0.0.0-20171010151810-6e5ba93211ea
+	github.com/im7mortal/kmutex v1.0.1
```

Comment:
Was this approved, didn't see an email to juju-crew.

Reply:
It's only a trivial mutex implementation

Reply:
decided to move this to juju/mutex